CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.0.6
+
+**Chore** - Backend binaries compiled with latest go version 1.19.4
+**Chore** - Backend grafana dependencies updated to latest version
```

Comment:
Clickhouse dependency was also updated, but was not released yet, shall we also add it to changelog?